### ISSUE TITLE
aaaa

### ISSUE THREAD

**Issue by Huseyin34** [google/spatial-media#233]
<!DOCTYPE html>
<html>
<head>
	<title>5 Panoramic</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0" />
	<meta name="apple-mobile-web-app-capable" content="yes" />
	<meta name="apple-mobile-web-app-status-bar-style" content="black" />
	<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
	<meta http-equiv="x-ua-compatible" content="IE=edge" />
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css">
	<script src ="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src ="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.6/umd/popper.min.js"></script>
	<script src ="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js"></script>
	<style>
		@-ms-viewport { width:device-width; }
		@media only screen and (min-device-width:800px) { html { overflow:hidden; } }
		html { height:100%; }
		body { height: 100%; overflow: hidden; margin: 0; padding: 0; }
	</style>
</head>
<body>

<script type="text/javascript">
	$(window).on('load', function () {
	if($(window).width() > 812)
		$('#myModal').modal('show');
	});
</script>

<div class="container">
	
	<div class="modal modalgreen fade" id="myModal">
		<div class="modal-dialog">
			<div class="modal-content">

				
				<div class="modal-header">
					<h4 class="modal-title">5</h4>
					<button type="button" class="close" data-dismiss="modal">&times;</button>
				</div>

				
				<div class="modal-body">
					Bu panoramik çıktıyı tekrar görüntülemek için panoramik çıktı klasöründeki RunMe.exe dosyasına çift tıklayın.
				</div>

				
				<div class="modal-footer">
					<button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
				</div>

			</div>
		</div>
	</div>
</div>

<script src="Data/tour.js"></script>

<div id="pano" style="width:100%;height:100%;">
	<noscript><table style="width:100%;height:100%;"><tr style="vertical-align:middle;"><td><div style="text-align:center;">ERROR:<br/><br/>Javascript not activated<br/><br/></div></td></tr></table></noscript>
	<script>
		embedpano({swf:"Data/tour.swf", xml:"Data/tour.xml", target:"pano", html5:"prefer", mobilescale:1.0, passQueryParameters:true});
	</script>
</div>

</body>
</html>
